.env

```
@@ -1,5 +1,5 @@
 # For test end ponint which depends on the login
-ACCESS_TOKEN="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjoiNjdjMDYzOWQ4MzZlZThiZTcwNjE5MGE5Iiwicm9sZSI6ImFkbWluIiwiaWF0IjoxNzYwMTk5MzQ5fQ.E61wk2cgsIEC0IihKE8vVS999r0OWh88WIE3e94F_6o"
+ACCESS_TOKEN="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjoiNjhlY2I3N2I3ZmVlNjhhNGMyZWI2ZTU5Iiwicm9sZSI6InVzZXIiLCJpYXQiOjE3NjA4NTc5MDR9.OGuuUIDNiWDQTog_yornJffdkOdVzYc8uiaExWd7kkQ"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)